style.css

```diff
@@ -997,3 +1014,128 @@ button {
     margin-top: 152px;
   }
 }
+
+/* POPUP SECTION */
+.popup {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  position: absolute;
+  overflow: auto;
+  padding: 16px;
+  gap: 12px;
+  width: 93%;
+  height: fit-content;
+  min-width: 90vw;
+  min-height: 100vh;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  box-sizing: border-box;
+  margin: 20px auto;
```

Comment:
- [ ] Please fix the positioning, margins, and width values for all elements of the popup card and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/189016087-1761dc4b-f85a-4e48-9afc-465fd17ef54a.png)  | ![image](https://user-images.githubusercontent.com/68796232/189015870-eb4f77e0-15d2-4ac8-959e-7daa6a4209ea.png) |